sympy/solvers/bivariate.py

```diff
@@ -17,7 +17,8 @@
 
 
 def _filtered_gens(poly, symbol):
-    """process the generators of ``poly``, returning the set of generators that
+    """
+    process the generators of ``poly``, returning the set of generators that
     have ``symbol``.  If there are two generators that are inverses of each other,
```

Comment:
Does adding a newline there actually do anything to the rendered output?

Reply:
sorry, for that I didn't get this properly . we need to add new line after and before each doc string as per PEP 257 multi line docstrings , right ??

Reply:
The referenced issue is about doctest examples not showing up in the web docs. There just needs to be some space around the doctest examples here:
https://github.com/sympy/sympy/blob/9e8f62e059d83178c1d8a1e19acac5473bdbf1c1/sympy/solvers/ode/systems.py#L2053-L2056